wgengine/magicsock/magicsock_test.go

```diff
@@ -120,6 +123,7 @@ type magicStack struct {
 	tun        *tuntest.ChannelTUN // tuntap device to send/receive packets
 	tsTun      *tstun.TUN          // wrapped tun that implements filtering and wgengine hooks
 	dev        *device.Device      // the wireguard-go Device that connects the previous things
+	tsIP       chan netaddr.IP     // buffered, guaranteed to yield at least 1 value
```

Comment:
what's the value?

Reply:
Replaced this with using ipnstate to pull the current Tailscale IP(s) out of the network engine. Also means you can get the IP multiple times without weirdness, which is nice.